core/commands/files/files.go

```diff
@@ -664,11 +664,6 @@ stat' on the file or any of its ancestors.
 			return
 		}
 
-		var r io.Reader = input
-		if countfound {
-			r = io.LimitReader(r, int64(count))
```

Comment:
Hrm... Maybe you discovered a bug here. I think the copy below was supposed to be from 'r' instead of 'input'